bezier_library/src/bezier_grinding_surfacing.cpp

```diff
@@ -83,6 +83,9 @@ std::string BezierGrindingSurfacing::generateTrajectory(EigenSTL::vector_Affine3
   if (!keepUpperPartofDilatedMesh(input_meshes_[SURFACE_MESH], dilated_mesh, extrication_mesh))
     return "Failed to extract upper part of dilated mesh";
 
+  if(!removeIsolatedTrianglesFilter(extrication_mesh, 20))
+    return "Failed to filter extrication mesh for isolated triangle removing";
```

Comment:
Failed to filter isolated triangles in extrication mesh

20 looks like a good default value IMO.